lib/data/special_routes.yaml

```diff
@@ -244,4 +244,10 @@
       - "af07d5a5-df63-4ddc-9383-6a666845ebe9"
   :additional_routes:
     - :base_path: "/when-do-the-clocks-change.json"
-      :type: "exact"
\ No newline at end of file
+      :type: "exact"
+
+- :content_id: "5fc8b4bc-f899-4d4e-ae10-f9beeb172f50"
+  :base_path: "/favicon.ico"
+  :title: "Favicon"
+  :rendering_app: "frontend"
+  :description: "The favicon is the image displayed in locations such as the browser tabs."
```

Comment:
This route is already in the db though right? So do we need to add `:override_existing: true` ?

Reply:
Oh, good spot. Thanks! Fixed.